README.md

```diff
@@ -10,5 +10,54 @@ http APIs
 <br>
 <br>
 
-## How to Run ##
-Using SBT, run `docker:publishLocal` command to locally publish the docker image. To start the cluster, run `docker-compose up` command.
+## Getting Started
```

Comment:
I would explicitly call this section something like "Development" or "Getting Started with Development" since at some point getting started might mean how to use the tool itself